plan/rule_aggregation_elimination.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2018 PingCAP, Inc.
```

Comment:
https://github.com/pingcap/tidb/pull/7676 already did the same thing, this patch can be removed.

Reply:
It should be done when merge that into this. Or the test result will be strange.

Reply:
Ok, let's get https://github.com/pingcap/tidb/pull/7676 merged as soon as possible.

Reply:
So we should merge 7680 first...